src/emulator/storage/rules/manager.ts

```diff
@@ -15,39 +16,24 @@ export class StorageRulesManager {
   private _watcher = new chokidar.FSWatcher();
   private _logger = EmulatorLogger.forEmulator(Emulators.STORAGE);
 
-  private constructor(private _runtime: StorageRulesRuntime) {}
-
-  /**
-   * Constructs and initializes a {@link StorageRulesManager}. This must be done in a factory
-   * method in order to load the ruleset from the runtime asynchronously.
-   */
-  public static async createInstance(
-    rules: SourceFile | string,
-    runtime: StorageRulesRuntime
-  ): Promise<StorageRulesManager> {
-    const instance = new StorageRulesManager(runtime);
-    await instance.setSourceFile(rules);
-    return instance;
-  }
+  constructor(private _runtime: StorageRulesRuntime) {}
 
   get ruleset(): StorageRulesetInstance | undefined {
     return this._ruleset;
   }
 
-  get watcher(): chokidar.FSWatcher {
-    return this._watcher;
-  }
-
   /**
    * Updates the source file and, correspondingly, the file watcher and ruleset.
+   * @throws {@link FirebaseError} if file path is invalid.
```

Comment:
please fix, I believe this should just be  `@throws {FirebaseError}`

Reply:
Done.